### ISSUE TITLE
Search input box is misleading on narrow screens

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress-documentation#743]
When switching to narrow screens the search input shows up full width

<img width="700" alt="screen shot 2018-06-28 at 8 45 59 am" src="https://user-images.githubusercontent.com/2212006/42035070-040ef524-7ab0-11e8-9c46-f1d58d34c81c.png">

but it is not - it stays narrow, see the "clear" button

<img width="699" alt="screen shot 2018-06-28 at 8 46 10 am" src="https://user-images.githubusercontent.com/2212006/42035096-0eb9bd88-7ab0-11e8-9fbd-62f5456cd1e3.png">

thus if you try to click on the right side and type, you cannot, which is annoying

**Comment by jennifer-shehane** [comment]
🙈

**Comment by hkarn** [comment]
I would like to try and fix this as a first time contribution.

**Comment by hkarn** [comment]
I have 2 lines of CSS that will fix it but I am unable to build the docs. 
I have tried on several computers now with different versions of node including 8.4.0
I get Exit status 3221225725. Where do I ask for help about this and testing my solution?

**Comment by richdouglasevans** [comment]
Hey Håkan,

Nice one. If you can and do push a branch up to your fork of the Cypress docs I can clone it and build it on my side and let you know.

I'm using node v10 if that's any help.

**Comment by richdouglasevans** [comment]
Good stuff Håkan. I have cloned your repo and built your branch, here are screenshots.

## iPhone 6

At the iPhone 6 width the search bar now stretches across the width of the screen _and_ the clear button is now right-aligned.

Looks good 👍

<img width="419" alt="iphone6" src="https://user-images.githubusercontent.com/1855109/45610546-3ab8aa00-ba54-11e8-8099-1100bc22516d.png">

## Responsive

At a wider width -- `761px` but still not full desktop -- the search bar now stretches across the width of the screen _and_ the clear button is now right-aligned.

There appears to be some media styling that's keeping the Algolia search results container quite narrow. Might be worth looking to see if there's some styling of that container around the 760px mark.

<img width="802" alt="responsive" src="https://user-images.githubusercontent.com/1855109/45610559-45733f00-ba54-11e8-8eac-8a19670fb5b7.png">

## Regular Desktop

Behaviour remains as expected, all good 👍 

<img width="531" alt="desktop" src="https://user-images.githubusercontent.com/1855109/45610574-57ed7880-ba54-11e8-9a3c-f71ac216bd4c.png">

**Comment by hkarn** [comment]
Thank you! I will see if I can do something about the autocomplete container also later today.

I finally managed to build it on Vultr instance btw ... on like the 12 try with the same settings it just suddenly worked.

**Comment by richdouglasevans** [comment]
> But still will not build...

Darn, that's not making for a great experience for you 😞🤗🤔

Mmm... lemme list the steps I took with your repo from scratch about ~15 minutes ago. Maybe that'll help flush out something?

```shell
$ git clone https://github.com/hkarn/cypress-documentation.git hak
$ cd hak
$ npm i
$ git co issue-743-search-input-box-misleading
$ npm run build
$ npm start

> npm run start-server -- --no-validate


> cypress-documentation@0.0.0 start-server /Users/evansr1/hak
> node --stack-size=8192 ./index.js server --port 2222 "--no-validate"

disabling filter cleanup for environment development
in environment development site url is https://docs.cypress.io
NODE_ENV is: development
INFO  Start processing
INFO  Hexo is running at http://localhost:2222 . Press Ctrl+C to stop.
```

The build shouldn't be doing any URL validation so if it is then we'll need to look at that. Let me know.

**Comment by hkarn** [comment]
Smt like this is what I get on the Vultr instance (except the one time it worked), on my Windows computers it dies after right after INFO  Start processing

The log files are not really any more insightful then the console output...

[07:43:55] Finished 'pre:build' after 2.75 μs

> cypress-documentation@0.0.0 build /root/github/cypress-documentation
> node --stack-size=8192 ./index.js generate

disabling filter cleanup for environment development
in environment development site url is https://docs.cypress.io
NODE_ENV is: development
INFO  Start processing
Request to: http://api.jquery.com/next timed out. Ignoring this error and proceeding. Waited for 7038ms.
Request to: http://sinonjs.org/ timed out. Ignoring this error and proceeding. Waited for 9617ms.
Request to: https://api.jquery.com/nth-child-selector/ timed out. Ignoring this error and proceeding. Waited for 7654ms.
Request to: https://www.typescriptlang.org/ timed out. Ignoring this error and proceeding. Waited for 7655ms.
Request to: https://gitlab.com/cypress-io/cypress-example-docker-gitlab timed out. Ignoring this error and proceeding. Waited for 7654ms.
Request to: https://reactjs.org/ timed out. Ignoring this error and proceeding. Waited for 7655ms.
Request to: https://appveyor.com/ timed out. Ignoring this error and proceeding. Waited for 11271ms.
Request to: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures timed out. Ignoring this error and proceeding. Waited for 7176ms.
Request to: https://github.com/cypress-io/cypress-test-nested-projects timed out. Ignoring this error and proceeding. Waited for 7170ms.
Request to: https://github.com/cypress-io/cypress/blob/master/packages/server/lib/config.coffee timed out. Ignoring this error and proceeding. Waited for 7169ms.
Request to: https://yarnpkg.com/ timed out. Ignoring this error and proceeding. Waited for 7170ms.
Request to: https://www.google.com/chrome/browser/desktop/index.html timed out. Ignoring this error and proceeding. Waited for 10892ms.
Request to: https://github.com/cypress-io/cypress-tutorial-build-todo-starter failed.
Request to: https://www.npmjs.com/ failed.
Request to: https://vimeo.com/242961930#t=264s failed.
Request to: https://github.com/cypress-io/cypress-watch-preprocessor timed out. Ignoring this error and proceeding. Waited for 11970ms.
Request to: https://github.com/cypress-io/cypress-documentation timed out. Ignoring this error and proceeding. Waited for 8394ms.
Request to: https://www.npmjs.com/package/npx failed.
Request to: https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/stubbing-spying__window-fetch/cypress/integration/spy-stub-clock-spec.js failed.
Request to: https://github.com/adamgruber/mochawesome timed out. Ignoring this error and proceeding. Waited for 8634ms.
Killed
npm ERR! code ELIFECYCLE
npm ERR! errno 137
npm ERR! cypress-documentation@0.0.0 build: `node --stack-size=8192 ./index.js generate`
npm ERR! Exit status 137
npm ERR!
npm ERR! Failed at the cypress-documentation@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-09-17T07_44_57_014Z-debug.log

**Comment by hkarn** [comment]
Ok, I finally made it start building properly on my virtual server instance. I think the problem was missing libraries to compile all the packages in npm install, I think this could be the problem on my Windows computer too.

As for the issue the autocomplete is a bit tricky. I moved the with reduction for it one break-point down, it is better. I didn't see an apparent way to left align it though.

It will now go to 320px with at the 620px breakpoint instead of 1001px. (At the 500px breakpoint it is just barley too large). I might be possible to refine it more by adding more size for the box or more breakpoints but I am not sure it is necessary.
